pythonforandroid/bootstraps/common/build/jni/application/src/start.c

```diff
@@ -161,6 +157,9 @@ int main(int argc, char *argv[]) {
   Py_Initialize();
 
 #if PY_MAJOR_VERSION < 3
+  // Can't Py_SetPath in python2 but we can set PySys_SetPath, which must
```

Comment:
I'm wondering if we should just omit this change, since SSL works with python 3 now and crystax becomes increasingly unnecessary. I think the better plan is to dump Crystax very soon. OpenSSL is as far as I know the only part remaining that doesn't work with @inclement 's new python 3 recipe, and you're fixing this right here in this pull request.

Or is this also useful/necessary in some way for non-crystax builds?

Reply:
I think we cannot remove this line because we need that for the new version of python2 which also use the `python_bundle_dir` like python3.